test/appTests/organisationsTests/update.test.js

```diff
@@ -0,0 +1,74 @@
+jest.mock(
```

Comment:
I think this test should live in `test/appTests/organisationsTests/data/update.test.js` or `test/appTests/organisationsTests/data/organisationsStorage.update.test.js` (I think the latter is best).  This'll make it really clear at a glance what file is being tested without having to open it

This comment applies to the `add.test.js` above too